static/app/views/replays/replayTable/tableCell.tsx

```diff
@@ -30,43 +34,64 @@ export function SessionCell({
   const {projects} = useProjects();
   const project = projects.find(p => p.id === replay.project_id);
 
+  const replayDetails = {
+    pathname: `/organizations/${organization.slug}/replays/${project?.slug}:${replay.id}/`,
+    query: {
+      referrer,
+      ...eventView.generateQueryStringObject(),
+    },
+  };
+
   return (
-    <UserBadge
-      avatarSize={32}
-      displayName={
-        <Link
-          to={{
-            pathname: `/organizations/${organization.slug}/replays/${project?.slug}:${replay.id}/`,
-            query: {
-              referrer,
-              ...eventView.generateQueryStringObject(),
-            },
-          }}
-        >
-          {replay.user.display_name || ''}
-        </Link>
-      }
-      user={{
-        username: replay.user.display_name || '',
-        email: replay.user.email || '',
-        id: replay.user.id || '',
-        ip_address: replay.user.ip || '',
-        name: replay.user.username || '',
-      }}
-      // this is the subheading for the avatar, so displayEmail in this case is a misnomer
-      displayEmail={<StringWalker urls={replay.urls} />}
-    />
+    <Item>
+      <UserBadge
+        avatarSize={24}
+        displayName={
+          <MainLink to={replayDetails}>{replay.user.display_name || ''}</MainLink>
+        }
+        user={{
+          username: replay.user.display_name || '',
+          email: replay.user.email || '',
+          id: replay.user.id || '',
+          ip_address: replay.user.ip || '',
+          name: replay.user.username || '',
+        }}
```

Comment:
I think we do this in at least one other place... might be worth refactoring into a util fn.